<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delete_dialog</class>
 <widget class="QDialog" name="delete_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>حذف الكلمات</string>
  </property>
  <widget class="QPushButton" name="search_b">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>بحــــــث</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_b">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>260</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>حــذف</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_b">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>رجـــــــــوع</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_bar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>الكلــــــمة</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="result_bar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
